frontend/src/pages/MainPage.vue

```diff
@@ -109,7 +109,11 @@ export default {
      * Upon clicking a marker, mark it as selected
      * @param {Object} marker - the marker that was clicked, having id, lat and lng
      */
-    onMarkerClick(marker) {
+    async onMarkerClick(marker) {
+      // Briefly switch between selected properties
+      this.$store.commit("setSelectedCoordinates", { selected_coordinates: null });
+      await new Promise((resolve) => setTimeout(resolve, 10));
```

Comment:
Why is this here?

Reply:
This was a not so ideal workaround for another issue, fixed properly now